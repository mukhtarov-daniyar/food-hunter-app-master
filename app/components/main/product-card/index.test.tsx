import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import ProductCard from './index';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <ProductCard
        price={'price'}
        productId={'productId'}
        title={'title'}
        url={'title'}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
