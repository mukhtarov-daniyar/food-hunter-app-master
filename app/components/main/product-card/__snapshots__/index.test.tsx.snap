// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "#ffffff",
      "borderRadius": 38.46153846153846,
      "elevation": 5,
      "flexDirection": "row",
      "marginBottom": 28.846153846153847,
      "opacity": 1,
      "paddingHorizontal": 19.23076923076923,
      "paddingVertical": 19.23076923076923,
      "shadowColor": "black",
      "shadowOffset": {
        "height": -1.9230769230769231,
        "width": 5.769230769230769,
      },
      "shadowOpacity": 0.1,
      "shadowRadius": 13.461538461538462,
    }
  }
>
  <Image
    source={
      {
        "uri": "title",
      }
    }
    style={
      {
        "borderRadius": 38.46153846153846,
        "height": 192.30769230769232,
        "width": 250,
      }
    }
  />
  <View
    style={
      {
        "flex": 1,
        "justifyContent": "center",
        "marginLeft": 38.46153846153846,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 33.333333333333336,
          "fontWeight": "bold",
          "marginBottom": 19.23076923076923,
        }
      }
    >
      title
    </Text>
    <Text
      style={
        [
          {
            "fontSize": 29.166666666666668,
          },
        ]
      }
    >
      NaN
      $
    </Text>
  </View>
</View>
`;
